tensorflow/contrib/makefile/Makefile

```diff
@@ -910,12 +910,18 @@ $(PROTO_TEXT_OBJS) : $(PROTO_TEXT_PB_H_FILES)
 
 # Ensures we link CoreFoundation as it is used for time library when building
 # for Mac.
+ifeq ($(TARGET),OSX)
```

Comment:
Should we also include a check for `x86_64` like below? Or is this general for any CPU architecture running OSX?

Reply:
Someone with OSX knowledge would need to answer this. for the iOS compilation it does that check, so it's possible we need that.

Reply:
Let's add the check

Reply:
I added the check.